plugins/woocommerce/client/blocks/tests/e2e/tests/templates/template-customization.block_theme.spec.ts

```diff
@@ -13,6 +13,11 @@ import {
 import { CUSTOMIZABLE_WC_TEMPLATES } from './constants';
 
 test.describe( 'Template customization', () => {
+	test.skip(
+		( { wpCoreVersion } ) => wpCoreVersion <= 6.6,
+		'Skipping on WP 6.6 and below'
+	);
+
```

Comment:
Template tests (this and below) skipped as per p1742489145338069/1742483773.434129-slack-C02FL3X7KR6.